src/plugins/app-route/handler.js

```diff
@@ -0,0 +1,27 @@
+// @flow
```

Comment:
Might be worth adding a comment to this file explaining exactly what it's handling (language redirects and session tracking).